### PR TITLE
creating the java application

### PR DESCRIPTION
**Pull request by kirankolli82**
Creating the initial java application

### CODE REVIEW

File: src/main/java/com/cms/webservice/App.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ */
+public class App {
+
+    public String getGreeting() {
+        return "Hello world.";
+    }
+
+    public static void main(String[] args) {
+        System.out.println(new App().getGreeting());
+        System.out.println("Done!");
```
Comment (makusi-it) [inline src/main/java/com/cms/webservice/App.java:17]: Chage done to Welldonen

File: src/test/java/com/cms/webservice/AppTest.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package com.cms.webservice;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Test;
+
+public class AppTest {
+
```
Comment (makusi-it) [inline src/test/java/com/cms/webservice/AppTest.java:11]: Add one more test case

File: src/main/java/com/cms/webservice/App.java (hunk 1 above)
Comment (kirankolli82) [reply src/main/java/com/cms/webservice/App.java:17]: Thanks!

File: src/test/java/com/cms/webservice/AppTest.java (hunk 2 above)
Comment (kirankolli82) [reply src/test/java/com/cms/webservice/AppTest.java:11]: only one test needed

**Review by makusi-it — APPROVED** [review]
Looks good, approved

**Pull request merged**